### PR TITLE
1inch wallet

### PR DESCRIPTION
**Pull request by shoom3301**
### Description
1inch wallet Dapp browser

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.37.0",
+  "version": "1.37.1",
```
Comment (Adamj1232) [inline package.json]: Can you update version to `1.37.1-0.0.1` when you have a moment?
Current version is 1.37.1

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Thank you for your PR!
Looking forward to integrating 1Inch Wallet
A small change is requested

File: package.json (hunk 1 above)
Comment (shoom3301) [reply package.json]: Done

**Review by Adamj1232 — APPROVED** [review]
Nice, thank you!

File: src/modules/select/wallets/1inch.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        interface:
+          provider && getProviderName(provider) === '1inch'
+            ? createModernProviderInterface(provider)
+            : null
+      }
+    },
+    type: 'injected',
+    link: `https://wallet.1inch.io?url=${window.location.host}`,
+    installMessage: mobileWalletInstallMessage,
+    mobile: true,
```
Comment (Adamj1232) [inline src/modules/select/wallets/1inch.ts:32]: @shoom3301 wanted to make a note here, after testing locally it seems this wallet can work on desktop as well with a little tweaking. If you would like to add please submit a new PR with `desktop: true` and an update to provider. thanks again!!

**Pull request merged**